app_backup.py

```
@@ -0,0 +1,415 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)